aria/cli/core/aria.py

```diff
@@ -325,6 +339,46 @@ def __init__(self):
             default=defaults.SERVICE_TEMPLATE_FILENAME,
             help=helptexts.SERVICE_TEMPLATE_FILENAME)
 
+        self.show_full = click.option(
+            '-f',
+            '--full',
+            'mode',
+            is_flag=True,
+            flag_value='full',
```

Comment:
I think I don't like this approach as much at the previous one.
It seems like it simplifies the code a bit but makes things somewhat more confusing to the user - specifically the fact that only the last flag of a given "destination" (e.g. `mode`) takes hold.

If instead you have different destinations for these flags, it'd be easier to inform the user of misuse (even if for some reason the mutually-exclusive mechanism is problematic to use, you can always just manually validate the parameters in the command function itself)